qtop.py

```diff
@@ -2192,11 +2204,13 @@ class InvalidScheduler(Exception):
                             break
                     else:
                         output_partview_fp = display.show_part_view(file=output_fp, x=viewport.v_start, y=viewport.v_term_size)
+
                     cat_command = 'clear;cat %s' % output_partview_fp
                     _ = subprocess.call(cat_command, stdout=stdout, stderr=stdout, shell=True)
 
-                    read_char = wait_for_keypress_or_autorefresh(viewport, int(options.WATCH[0]) or KEYPRESS_TIMEOUT)
-                    control_qtop(viewport, read_char, cluster)
+                    read_char = wait_for_keypress_or_autorefresh(viewport, FALLBACK_TERMSIZE, int(options.WATCH[0]) or
+                                                                 KEYPRESS_TIMEOUT)
```

Comment:
this is the way your editor tells you: "you have far too long function name" ;-)